icu4c/source/i18n/unicode/nounit.h

```diff
@@ -32,77 +32,43 @@ U_NAMESPACE_BEGIN
  * @see NumberFormatter
  * @draft ICU 60
  */
-class U_I18N_API NoUnit: public MeasureUnit {
-public:
+namespace NoUnit {
     /**
      * Returns an instance for the base unit (dimensionless and no scaling).
      *
-     * @return               a NoUnit instance
+     * Since ICU 68, this function returns the same value as the default MeasureUnit constructor.
```

Comment:
See comment in the Java version: I think at least this one should go to `@draft ICU 68`.